README.md

```diff
@@ -30,6 +30,8 @@ The conference dataset hosted in this repository tracks the following informatio
 | [ZIO World](https://zioworld.com) | Ziverge | Yes | Yes | No | Never |
 | [ScalarConf](https://www.scalar-conf.com) | Software Mill | No | Yes | No | Never | 
 | [PyCon](https://pycon.org) | PyCon Board Committee | Pending | No | Yes | Keynote |
+| [JSConf](https://jsconf.com/) | Local Community | No | Yes | Partial | Yes | 
```

Comment:
If whether to sign the SEA is up to the local chapter, then I would put `Varies` in the column rather than `No`.

Reply:
As inferred from skimming through the resources very few would agree to sign the SEA, so I made it `No`. Do you want me to keep it as `varies`?

Reply:
An alternate would be to ask each local community, but that seems like far worse than just saying the answer `Varies` (depending on the chapter).

Reply:
I agree, JSConf is huge that it needs a separate spreadsheet of its own IMO.

Reply:
All right, then change the column to `Varies`.

Reply:
Done!